src/app/mutation-indicator/mutation-indicator.js

```diff
@@ -6,9 +6,13 @@ import React from 'react'
 const MutationIndicator = () => {
     const pendingMutations = useIsMutating()
     const message = pendingMutations
-        ? i18n.t('There are {{ pendingMutations }} pending mutations', {
-              pendingMutations,
-          })
+        ? pendingMutations === 1
+            ? i18n.t('There is {{pendingMutations}} pending change.', {
```

Comment:
true. thanks! I think I'll just leave it as is because it's more similar to what we are doing in other apps where the plurals are handled more dynamically: https://github.com/dhis2/line-listing-app/blob/dev/src/components/Layout/ChipBase.js#L47-L61
(though agree that it is a bit silly at the moment)